tests/preview_changed_file/deleted_in_working.fish

```diff
@@ -1,7 +1,5 @@
 mock git diff "echo \$argv"
-set file dir/file.txt
+set output (_fzf_preview_changed_file " D out.log")
 
-set output (_fzf_preview_changed_file " D $file")
-
-contains -- "| Unstaged |" $output && not contains "| Staged |" $output
```

Comment:
contains doesn't need `--` because by passing in the first argument, it would never interpret a line prefixed with`-` in $output as an option.